verticapy/learn/vmodel.py

```diff
@@ -4094,7 +4052,7 @@ def score(
                 self.test_relation,
                 self.cursor,
                 best_threshold=True,
-                nbins=1000,
+                nbins=nbins,
```

Comment:
The value of nbins becomes 10 times bigger. Is it necessary despite its performance penalty?

Reply:
Did we evaluate the performance of AUC = f(nbins). It seems that 10k is fast enough for most of the datasets.